include/RaytracerRules.hpp

```diff
@@ -0,0 +1,13 @@
+/*
+** EPITECH PROJECT, 2024
+** Raytracer
+** File description:
+** Rules header
+*/
+
+// This file handle all the important/common rules
+// of the project
+
+#pragma once
+
+#define VOID_COLOR Math::Vector3D(0,0,0)
```

Comment:
I'm not agree with the name of VOID_COLOR, maybe we can translate it by BLACK_COLOR ?

Reply:
No it's the color of the void, black is the color of the void but black is not the void. You understand ?

Reply:
I understand